plugins/org.jboss.tools.openshift.client/META-INF/MANIFEST.MF

```diff
@@ -20,6 +20,7 @@ Bundle-ClassPath: lib/openshift-restclient-java-5.1.0.Final.jar,
  lib/commons-io-2.1.jar,
  .
 Export-Package: com.openshift.internal.restclient;x-friends:="org.jboss.tools.openshift.test",
+ com.openshift.internal.restclient.authorization,
```

Comment:
btw. for great reusability the jobs within openshift.ui should be living in core. Their design (dependencies) prevent that unfortunately. IMHO not the best design. That's not the moment to address this though.